### DISCUSSION TITLE
Hidden collections [dReader]

### DISCUSSION THREAD

**Discussion by josip-fundl** [phantom/docs#152] [Help & Support] [answered]
### Summary

Hello 👋 

What makes collections hidden? Is there any criteria shared publicly?

We're using the Candy Machine program on our launchpad for comic NFTs and all 3 collections ended up in hidden.

Example NFT #1: https://solscan.io/token/H1JFV3kRfG2pB1YQv2kAX1E7CKmEtTN4ycFtCSx4FL9e
Example NFT #2: https://solscan.io/token/4Vgegugs31UKNUEPAdPL8zeyEgSB7g7RKVGFdzxhDA87
Example NFT #3: https://solscan.io/token/8MeGtFzNQGPxgUxgaDuf9E7qJoJcf9TdVT7Pwmi1tg2Q

All 3 (from 3 different collections) were hidden.

All 3 candy machines share the same creator, our backend's treasury: `FXj8W4m33SgLB5ZAg35g8wsqFTvywc6fmJTXzoQQhrVf`
Could our "creator" wallet be flagged as a spammer?

Any help troubleshooting would be greatly appreciated

### Steps to Reproduce

-

### Phantom Version

latest

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by josip-fundl** [comment]
@adamdelphantom

**Reply by raulriera** [reply, marked as answer]
Hey, from our checks it seems like Example 1 and 3 have not been a spam for a while (Dec 1 and Dec 25). For Example 2 we have unmarked it. 

Are you still seeing them in the hidden folder? If you are, did you ever manually move them there? User reports are also considered to mark things as spam or not

**Comment by jim4067** [comment]
I was wondering why my mints weren't showing up at first.  @bfriel please help

**Comment by codinpsycho** [comment]
We faced the same problem at Shyft.to with our Brainfood series, we were minting video and blog content as cNFTs that was going into hidden collections.

**Reply by raulriera** [reply]
Hey Ishkaran, mind reaching out to support about your case? Given that they have been largely airdropped to the community it's possible they have been a ton of user reports. We can look further in another thread

**Comment by mikl10** [comment]
It would be great to solve that 👍

**Comment by topicanica** [comment]
I would greatly appreciate if you could look into it! And so would the whole team at dReader, since they've been working very hard on this.

**Comment by EventMerge** [comment]
I've been trying to determine what makes my nfts hidden as well on mint, has there been a resolution on this? would be helpful to know
